frontend/homeTabs.jsx

```diff
@@ -37,38 +42,50 @@ export function DiagnosticsTab() {
   const heartRateDiagnostics = sampleData.map((user) => user.heartRate);
 
   return (
-    <View style={styles.container}>
-      <Text>Heart Rate Analysis</Text>
-      <LineChart
-        data={{ labels: daysOfTheWeek, datasets: [{ data: heartRateDiagnostics }] }}
-        height={200}
-        width={Dimensions.get('window').width - 60}
-        fromZero
-        withShadow={false}
-        chartConfig={{
-          backgroundGradientFrom: '#A6CDF0',
-          backgroundGradientTo: '#CFDFF7',
-          color: (opacity = 1) => `rgba(0,0,0, ${opacity})`,
-        }}
-      />
-    </View>
+    <PageView>
+      <PageTitle>Diagnostics</PageTitle>
+      <CardView>
+        <LinearGradient colors={['#F9F6FF', '#CFDFF7']} style={{ padding: 20, paddingTop: 0, borderRadius: 10 }}>
+          <CardTitle>Heart Rate Analysis</CardTitle>
+          <LineChart
+            data={{ labels: daysOfTheWeek, datasets: [{ data: heartRateDiagnostics }] }}
+            height={200}
+            width={Dimensions.get('window').width - 60}
+            fromZero
+            withShadow={false}
+            chartConfig={{
+              backgroundGradientFrom: '#A6CDF0',
+              backgroundGradientTo: '#CFDFF7',
+              color: (opacity = 1) => `rgba(0,0,0, ${opacity})`,
+            }}
+            style={{ borderRadius: 10 }}
+          />
+        </LinearGradient>
+      </CardView>
+    </PageView>
   );
 }
 
 export function HomeTab({ navigation }) {
   return (
-    <PageView>
-      <PageTitle home>Hello, Person</PageTitle>
+    <PageView center>
+      <PageTitle>Hello, Person</PageTitle>
       {/*
             The Home tab contains a Button titled "Start"
             that navigates to the Start Screen. Because the
             corresponding route is a Stack.Screen in the parent
             Navigator (in App.js), the bottom tab bar is not displayed
             */}
+      <CardView home>
+        <LinearGradient colors={['#F9F6FF', '#CFDFF7']} style={{ padding: 20, paddingTop: 8, borderRadius: 10 }}>
+          <ButtonText>Last Night,</ButtonText>
+          <CardTitle center>2432 minutes</CardTitle>
+        </LinearGradient>
+      </CardView>
       <Feather.Button
         name="moon"
-        color="#000"
-        backgroundColor="#fff"
+        color="black"
+        backgroundColor="white"
         onPress={() => navigation.navigate('Start')}
         size={50}
       />
```

Comment:
The moon icon here is ever so slightly off center

<img width="478" alt="image" src="https://user-images.githubusercontent.com/3270324/103712978-01cbd880-4f70-11eb-9a6e-8002c0c016d5.png">